app/views/app_groups/_applications.html.slim

```diff
@@ -4,7 +4,9 @@ h4.mb-3 All Applications
   table.table.table-hover.table-sm
     thead
       tr.d-flex
-        - if @show_log_and_cost_col || !@app_group.disable_app_tps
+        - if @show_log_and_cost_col && !@app_group.disable_app_tps
```

Comment:
I guess we need to align the values in `tbody` as well if we change the alignment of label columns.

Reply:
updated ya @Abhinav1299